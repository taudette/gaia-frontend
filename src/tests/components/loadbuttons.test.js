import React from 'react';
import { shallow } from 'enzyme';
import LoadButtonsComponent from '../../components/Video';
import data from '../fixtures/loadButtons';

test('should render LoadButtonsComponent without props', () => {
  const wrapper = shallow(<LoadButtonsComponent  />)
  expect(wrapper).toMatchSnapshot();
})

test('should render LoadButtonsComponent with props', () => {
  const wrapper = shallow(<LoadButtonsComponent addVideos={data.addVideos} goToTop={data.goToTop} endIndex={data.index} listLength={data.listLength} fetching={data.loading} />)
  expect(wrapper).toMatchSnapshot();
})
