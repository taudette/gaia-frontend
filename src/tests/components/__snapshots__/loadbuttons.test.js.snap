// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render LoadButtonsComponent with props 1`] = `
<Col
  bsClass="col"
  componentClass="div"
  lg={3}
  md={4}
  sm={6}
  xs={12}
>
  <li
    className="tile"
  />
</Col>
`;

exports[`should render LoadButtonsComponent without props 1`] = `
<Col
  bsClass="col"
  componentClass="div"
  lg={3}
  md={4}
  sm={6}
  xs={12}
>
  <li
    className="tile"
  />
</Col>
`;
